src/components/TodoItem/TodoItem.tsx

```diff
@@ -0,0 +1,56 @@
+/* eslint-disable jsx-a11y/label-has-associated-control */
+import React from 'react';
+import { Todo } from '../../types/Todo';
+import cn from 'classnames';
+
+interface Props {
+  todo: Todo;
+  deleteTodo: (todoId: number) => void;
+  isDeleting: boolean;
+  isTemporary?: boolean;
```

Comment:
```suggestion
  isProcessing: boolean;
```
No need to add such flag as `isTemporary` it should be just regular todo.
And `isDeleting` is a specific name, in the next task you will another functionality when you need to show loader so you can name as `isLoading` or `isProcessing`